mass_mailing_sending_queue/README.rst

```diff
@@ -0,0 +1,89 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==========================
+Mass mailing sending queue
+==========================
+
+This module adds a sending queue when mass mailing 'Send to All' button is
+clicked. This is an aditional queue, there is another one for sending emails.
+
+Configuration
+=============
+
+There is a system parameter to control how many emails are created in each
+cron iteration: 'mail.mass_mailing.sending.batch_size'. By default, 500
+
+
+Usage
+=====
+
+When 'Send to All' button is clicked at mass mailing form, all 'mail.mail'
+and 'mail.mail.statistics' objects are created. This process shall take
+so much time if recipient list is 10k+ and the famous
+"Take a minute to get a coffee, because it's loading..." text appears.
+
+With this new queue the user will see that mass mailing is in 'Sending' state
+until all emails are sent or failed. Screen will return to mass mailing form
+quickly after 'Send to All' button is clicked.
+
+In 'Mass mailing' form there is a new tab 'Sending tasks' for checking
+sendings history.
```

Comment:
@Yajo lol: I couldnot compete with @pedrobaeza for the python part so I specialized myself :p 

"In 'Mass mailing' form, a new tab "Sent mail" has been added where the user can check the Sent mails history."